mlir/tools/mlir-miopen-lib/mlir-miopen-lib.hpp

```diff
@@ -2,20 +2,27 @@
 
 #define MLIRMIOPEN_VERSION_FLAT 0
 
+enum {
+  EMlirSuccess = 0,
+  EMlirInvalidParam = -1,
+  EMlirInvalidModule = -2,
+  EMlirBuildFailure = -3
+};
+
 /*! @brief The MLIR handle used for lowering and code generation
  */
 typedef void *MlirHandle;
 
 /*! @brief Create the MLIR handle according to options string
- *  @param options A comma seperated string
+ *  @param options A comma seperated string??
  *  @return        MLIR handle
  */
 extern "C" MlirHandle CreateMlirHandle(const char *options);
 
 /*! @brief Lower the MLIR module to c++ code
- *  @param handle MLIR handle
+ *  @param handle   MLIR handle
```

Comment:
redundant  `??` and `  `

Reply:
well, it's not a comma separated list so maybe we should just change the comment :)